add-screenshot/index.md

```diff
@@ -0,0 +1,28 @@
+## Adding screenshots to my game page
+
+There's a distinct possibility that screenshots are the single most vital element of your game pages, in terms of catching the attention of people who have already been enticed by your thumbnail, or who are visiting via a direct link to your game page. Many people will simply skip over a downloadable game with no screenshots.
```

Comment:
should we say images instead of screenshots? "There's a distinct possibility that images are the single ..."